### ISSUE TITLE
About Circle Loss

### ISSUE THREAD

**Issue by finger-monkey** [layumi/Person_reID_baseline_pytorch#263]
hi,sir. When I use the Circle Loss to trian model:
python train.py  --name ft_resNet50_circle --circle ,I encountered the following problem:

![image](https://user-images.githubusercontent.com/52326609/115647304-19c83c00-a356-11eb-8e01-d25c387b14b6.png)
chould you share me how to solve this problem?Thank you!
All the best.

**Comment by CynicalHeart** [comment]
Pytorch version should >= 1.4

**Comment by finger-monkey** [comment]
> Pytorch version should >= 1.4

that's right.Thank you!

**Comment by finger-monkey** [comment]
![image](https://user-images.githubusercontent.com/52326609/116210792-c7cc4f80-a775-11eb-9fa6-b4c36046464b.png)

Now my torch version=1.4.0,when I test the model ,I encounted this problem.

**Comment by finger-monkey** [comment]
> Pytorch version should >= 1.4

Now my torch version=1.4.0.It can be trained normally,but when I test the model , encounted this problem.
![image](https://user-images.githubusercontent.com/52326609/116213845-b5074a00-a778-11eb-8d80-20523dd13de6.png)
could you share me how to solve this problem?

**Comment by CynicalHeart** [comment]
> ![image](https://user-images.githubusercontent.com/52326609/116210792-c7cc4f80-a775-11eb-9fa6-b4c36046464b.png)
> 
> Now my torch version=1.4.0.It can be trained normally,but when I test the model , encounted this problem.


Sorry, I didn't encounter this problem. In line 13, **"Your PyTorch installation may be too old"**, I think you should update the `conda` overall environment, and I use the `pytorch` version of the version 1.7.0. BTW,  i found your problem in the load model. Make sure that the training and testing models are version consistent

**Comment by finger-monkey** [comment]
oh，may be you are right. In my issue,the training and testing models are version not consistent. Thanks for your patience.

**Comment by CynicalHeart** [comment]
> oh，may be you are right. In my issue,the training and testing models are version not consistent. Thanks for your patience.

You're welcome. I also have a lot of problems adjusting the parameters. We need to help each other. 😃
